src/main/resources/docs/description/CommentsIndentation.md

```diff
@@ -1,6 +1,6 @@
 Since Checkstyle 6.10
 
-Controls the indentation between comments and surrounding code. Comments are indented at the same level as the surrounding code. Detailed info about such convention can be found [ here][here]
+Controls the indentation between comments and surrounding code. Comments are indented at the same level as the surrounding code. Detailed info about such convention can be found  here
```

Comment:
I think the [here] link got lost, right?

Reply:
Before the pr link where broken: They were `[Description][Description]` instead of `[Description](actual link)`
This was because of a buggy implementation on remark library (last commit in 2016).
Now I changed to flexmark and it behaves correctly.